### DISCUSSION TITLE
Object not displaying in browser.

### DISCUSSION THREAD

**Discussion by LevelMarketing** [google/model-viewer#2080] [Q&A]
I can't seem to display a GLB object within my web browser using model-viewer. I have the object hosted on AWS and am using webflow as a test development site with the following code:

**Header**
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>

**Body**
<--model-viewer 
                  src="https://l4test.s3.eu-central-1.amazonaws.com/JCI_KNX_Demobox_Sensor_AR_004_Selected.glb"
                  ios-src="https://l4test.s3.eu-central-1.amazonaws.com/JCI_KNX_Demobox_Sensor_AR_004_Selected.usdz"
                  auto-rotate camera-controls ar>
<--/model-viewer>

The '--' has been added so the code can be read and is not on the dev site. 

Please find a link to the site here: https://modelviewer.webflow.io

The ios src appears to work correctly and I can view the object using the iPhone AR viewer but the original src="" isn't pulling through anything. I've had a look through the discussions as this would seem like a common problem but couldn't find anything. Would the issue likely be with the code or the GLB file?

**Comment by elalish** [comment]
Do you get any errors in the console? Have you dropped your model into our [editor](https://modelviewer.dev/editor/) to check for validation errors? Does the GLB load in other web viewers?

**Reply by LevelMarketing** [reply]
Ah it would appear [there's an issue](https://assets.website-files.com/603e6b66bae24b1fcdc5c332/603e73731e9b75250b7d32a2_Screenshot%202021-03-02%20at%2017.18.10-p-2000.png) with access. Is there a recommended hosting service? 

I've just dropped it into the editor and get the [following validation errors.](https://assets.website-files.com/603e6b66bae24b1fcdc5c332/603e72af3cdb13d81ab6d780_Screenshot%202021-03-02%20at%2017.13.29.png)

**Reply by elalish** [reply]
No, you can host on anything that lets you set appropriate CORS headers. Github pages does this by default, which is what we use.

**Reply by LevelMarketing** [reply]
Sorry, that went over my head. Is the issue with the server the GLB file is stored on or is it with Webflow?

**Reply by elalish** [reply]
Yes, CORS headers need to be set on the server. It's a security measure that allows you to restrict what domains can access these files, but it defaults to very restrictive (hence the security).
